schema/migration-2-0035-20240104.sql

```diff
@@ -0,0 +1,25 @@
+-- Persistent generated migration.
+
+CREATE FUNCTION migrate() RETURNS void AS $$
+DECLARE
+  next_version int ;
+BEGIN
+  SELECT stage_two + 1 INTO next_version FROM schema_version ;
+  IF next_version = 35 THEN
+    EXECUTE 'ALTER TABLE "tx_metadata" ALTER COLUMN "json" TYPE VARCHAR' ;
```

Comment:
Would this make sense to be `text` (no limit)?

Reply:
guess persistent converts the type `Text` to psql type `VARCHAR`?

Reply:
varchar and text are the same under the hood https://stackoverflow.com/questions/4848964/difference-between-text-and-varchar-character-varying